src/main/java/net/helpscout/api/ApiClient.java

```diff
@@ -79,57 +84,80 @@
 
 public class ApiClient {
```

Comment:
I realise you couldn't have possibly refactored all of this, but uff this is a refactoring job for couple of days :/ Maybe we could at least spawn a trello card that we need to do something about this in the future?